src/utilities/CMakeLists.txt

```diff
@@ -31,8 +30,14 @@ if (TRACY_ENABLE)
     GIT_PROGRESS TRUE
   )
   FetchContent_MakeAvailable(tracy)
-  target_link_libraries(${PROJECT_NAME} PUBLIC TracyClient)
-  target_compile_options(${PROJECT_NAME} PUBLIC -DTRACY_NO_CODE_TRANSFER=1)
+  target_link_libraries(${PROJECT_NAME} INTERFACE TracyClient)
+  target_compile_options(${PROJECT_NAME} INTERFACE -DTRACY_NO_CODE_TRANSFER=1)
+endif()
+
+if(EMSCRIPTEN)
+    set(TBB_GIT_TAG 8db4efacad3b8aa6eea62281a9e8444e5dc8f16a)
```

Comment:
What's this?

Reply:
Oh, this is the TBB on wasm thing that I accidentally commited, will remove them.